frontend/src/libs/exportUtils.ts

```diff
@@ -41,6 +41,7 @@ function wageChunksMatchPositionAndSession(
     if (wageChunks.length === 1) {
         const chunk: WageChunk = wageChunks[0];
         if (
+            // shouldn't we also check chunk.hours === assignment.hours? Also sum of hours === assignment.hours when there are multiple chunks?
```

Comment:
@siefkenj Do we need to check chunk.hours === assignment.hours? Also sum of hours === assignment.hours when there are multiple chunks?

Reply:
Yes, I think we probably should...

Reply:
Remove this comment and make an issue for it.

Reply:
moved the issue to #504